tests/hello.txt

```diff
@@ -0,0 +1 @@
+Hello, World!
```

Comment:
Where should I put this file?

Reply:
*ideally* nowhere :D but I'm not sure if 

```rust
let dir = TempDir::new("my_directory_prefix").unwrap();
std::fs::write(dir.join("hello.txt"), "Hello, World!").unwrap();
assert_eq!(std::fs::read(dir.join("hello.txt")).unwrap(), "Hello, World!");
```

is already possible today

Reply:
Jmmm... that would mean adding tempdir as a dev dependency, is that ok?

Reply:
yea, let's not do that :)

Reply:
I'm going to keep that file temporarily until the `(mk|rm)dir` shims are implemented. I'll add a fixme somewhere.